recipe/build.sh

```diff
@@ -45,6 +45,12 @@ else
     LLAMA_ARGS="${LLAMA_ARGS} -DLLAMA_BLAS=OFF"
 fi
 
+if [[ ${x86_64_opt:-} = "v3" ]]; then
```

Comment:
do we need to change the LLAMA_AVX flags below as well?

Reply:
You can but they do not need to be set. When the LLAMA_* flags are set, cmake includes the associated compiler flags (`-mavx`, etc) when calling the compiler. The change to `-march` accomplishes the same results since `x86-64-v3` enables the same instruction sets and macros as the instruction set specific compiler flags.

Reply:
Setting march=x86-64-v3 is sufficient from my understanding.

Reply:
ok, it's a bit confusing for future people not familiar with it so would prefer it to be removed but that's nonfunctional